.vscode/settings.json

```diff
@@ -133,7 +133,8 @@
         "random": "cpp",
         "thread": "cpp",
         "variant": "cpp",
-        "any": "cpp"
+        "any": "cpp",
+        "future": "cpp"
```

Comment:
This is what happens if you open any new file and save it in VSCode, so doing it for folks here